disperser/cmd/controller/main.go

```diff
@@ -275,6 +277,34 @@ func RunController(ctx *cli.Context) error {
 		return fmt.Errorf("failed to start dispatcher: %v", err)
 	}
 
+	if config.ServerConfig.EnableServer {
+		var paymentAuthorizationHandler *payments.PaymentAuthorizationHandler
+		if config.ServerConfig.EnablePaymentAuthentication {
+			paymentAuthorizationHandler = payments.NewPaymentAuthorizationHandler()
+		}
+
+		grpcServer, err := grpcserver.NewServer(
+			c,
+			config.ServerConfig,
+			config.AwsClientConfig,
+			config.DisperserKMSKeyID,
+			logger,
+			metricsRegistry,
+			paymentAuthorizationHandler)
+		if err != nil {
+			return fmt.Errorf("create gRPC server: %w", err)
+		}
+
+		go func() {
+			logger.Info("Starting controller gRPC server", "port", config.ServerConfig.GrpcPort)
+			if err := grpcServer.Start(); err != nil {
+				logger.Error("gRPC server failed", "error", err)
+			}
```

Comment:
Perhaps a good time to panic? If the gRPC server is not online, the disperser as a whole is probably in a broken state. (It's possible you were following examples from elsewhere that don't handle this well.)